lib/puppet_x/puppetlabs/scheduled_task/task.rb

```diff
@@ -372,6 +386,18 @@ def self.folder_path_from_task_path(task_path)
     path.empty? ? ROOT_FOLDER : path
   end
 
+  def self.create_folder(path)
+    service = task_service
+    begin
+      service.GetFolder(path)
+    rescue WIN32OLERuntimeError => e
+      unless Error.is_com_error_type(e, Error::ERROR_FILE_NOT_FOUND)
+        raise Puppet::Error.new( _("GetTask failed with: %{error}") % { error: e }, e )
+      end
+      service.GetFolder("\\").CreateFolder(path)
```

Comment:
Do we want to trap errors for this?

Reply:
Probably not

Reply:
I'm leaning towards "no" initially, but I'm open to it.